lib/connect.js

```diff
@@ -47,7 +47,7 @@ function openFrames(vhost, query, credentials, extraClientProperties) {
   if (!vhost) vhost = '/';
   else vhost = QS.unescape(vhost);
 
-  var query = query || {};
+  query = query || {};
```

Comment:
query is provided as a function parameter but was unused because it was redeclared and masked by `var`. I removed the var as the was presumably the original intent. The tests still pass.